panel/widgets/slider.py

```diff
@@ -594,8 +594,8 @@ class DateRangeSlider(_RangeSliderBase):
     ... )
     """
 
-    value = param.Tuple(default=(None, None), length=2, doc=
-        """The selected range as a tuple of values. Updated when one of the handles is
+    value = param.DateRange(default=None, doc="""
+        The selected range as a tuple of values. Updated when one of the handles is
```

Comment:
Don't you also have to update `value_throttled`?

Reply:
You're right I missed it thanks for the catch! I'll fix that.

Reply:
I had not yet changed it, I'll submit another PR.

Reply:
Sorry!